tests/test_generate_data.py

```diff
@@ -84,31 +84,180 @@ def test_create_pt(self, mock_file):
             check_top_dict['links'][int(link_num)] = data_dict
         self.assertEqual(topology_dict, check_top_dict)
 
-    def test_create_bw_info_yue(self):
+    @patch('builtins.open', new_callable=mock_open, read_data=json.dumps({
```

Comment:
Is this variable still in the file itself?

Reply:
It shouldn't have been. I forgot to push the changes